GroceriesApp/GroceryappDesign.sql

```diff
@@ -0,0 +1,158 @@
+CREATE DATABASE GroceriesAppdb;
+Use GroceriesAppdb;
+
+--creating table
+
+DROP TABLE UserDetails
+CREATE TABLE  UserDetails(
+    RegisterId INT IDENTITY(1,1),
+    FirstName VARCHAR(20) NOT NULL,
+    LastName VARCHAR(20) NOT NULL,
+    Email VARCHAR(40) , -- Unique email
+    [Password] VARCHAR(20) NOT NULL,
+    ConfirmPassword VARCHAR(20) NOT NULL,
```

Comment:
you can remove confirmpassword  column
No need of having confirmpassword column we can validate from UI